src/gui/hud/SecondaryInventory.cpp

```diff
@@ -558,7 +558,8 @@ bool SecondaryInventoryHud::dragEntity(Entity * io, const Vec2s & pos) {
 		}
 	}
 	
-	return false;
+	Set_DragInter(io);
```

Comment:
> The thing is, they weren't there before.

Well, they weren't there in that function, but as far as code flow is concerned, they were reached before.

> But what purpose does RemoveFromAllInventories serve?

Removes all references to the entity from the inventories of other entities. Unless i'm missing something, it shouldn't be needed because the entity should only ever be in one inventory, it must be in the SecondaryInventory or we wouldn't get here, and it is removed from that inventory in the code above.